doc/combining.rst

```diff
@@ -13,6 +13,8 @@ Combining data
 
 * For combining datasets or data arrays along a dimension, see concatenate_.
 * For combining datasets with different variables, see merge_.
+# For combining datasets with different coordinates or missing values,
```

Comment:
use `*` at the start for consistency

Maybe say "datasets or data arrays"?

Also, maybe "different indexes" instead of "different coordinates"?